package.json

```diff
@@ -34,42 +35,47 @@
     "eslint-plugin-import": "^2.17.3",
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-react": "^7.13.0",
+    "file-loader": "^3.0.1",
+    "image-webpack-loader": "^5.0.0",
     "jest": "^24.8.0",
     "jest-css-modules": "^2.0.0",
     "redux-mock-store": "^1.5.3"
   },
   "dependencies": {
     "@babel/core": "^7.4.5",
     "@babel/polyfill": "^7.4.4",
-    "@fortawesome/fontawesome-svg-core": "^1.2.18",
```

Comment:
Hello @Amoswachira and @Bakley ,
Why have you removed the font-awesome modules?
Since we are using them in our application, I think they should not be done away with.

Reply:
@Ogutu-Brian  there is another font awesome CND link include in the index.html so the font awesome icons will still load

Reply:
Okay